src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -47,25 +48,28 @@
  */
 @SuppressWarnings("deprecation")
 @Name("Message")
-@Description("The (chat) message of a chat event, the join message of a join event, the quit message of a quit event, or the death message on a death event. This expression is mostly useful for being changed.")
+@Description("The (chat) message of a chat event, the join message of a join event, the quit message of a quit event, or the death message on a death event, or the unknown command message on a unknown command event. This expression is mostly useful for being changed.")
 @Examples({"on chat:",
-		"	player has permission \"admin\"",
-		"	set message to \"&c%message%\"",
+			"\tplayer has permission \"admin\"",
+			"\tset message to \"&c%message%\"",
 		"",
 		"on first join:",
-		"	set join message to \"Welcome %player% to our awesome server!\"",
+			"\tset join message to \"Welcome %player% to our awesome server!\"",
 		"",
 		"on join:",
-		"	player has played before",
-		"	set join message to \"Welcome back, %player%!\"",
+			"\tplayer has played before",
+			"\tset join message to \"Welcome back, %player%!\"",
 		"",
 		"on quit:",
-		"	set quit message to \"%player% left this awesome server!\"",
+			"\tset quit message to \"%player% left this awesome server!\"",
 		"",
 		"on death:",
-		"	set the death message to \"%player% died!\""})
-@Since("1.4.6 (chat message), 1.4.9 (join & quit messages), 2.0 (death message)")
-@Events({"chat", "join", "quit", "death"})
+			"\tset the death message to \"%player% died!\"",
+		"",
+		"on unknown command:",
+			"\tset unknown command message to \"This command does not exist.\""})
+@Since("1.4.6 (chat message), 1.4.9 (join & quit messages), 2.0 (death message), INSERT VERSION (unknown command message)")
+@Events({"chat", "join", "quit", "death", "unknown command message"})
```

Comment:
This should use the event's name `Unknown Command`
```suggestion
@Events({"chat", "join", "quit", "death", "unknown command"})
```